cmd/clusterctl/client/cluster/mover_test.go

```diff
@@ -478,10 +502,11 @@ func Test_objectMover_move(t *testing.T) {
 
 				err := csFrom.Get(ctx, key, oFrom)
 				if err == nil {
-					t.Errorf("%v not deleted in source cluster", key)
-					continue
-				}
-				if !apierrors.IsNotFound(err) {
+					if oFrom.GetNamespace() != "" {
```

Comment:
```suggestion
					if err== nil && oFrom.GetNamespace() != ""
```

Reply:
I don't think I can quite do this. I tried it at first. If err is nil and there's no namespace for the object, it won't meet this condition and will fail in the condition below with the error `mover_test.go:508: error = <nil> when checking for /externalTest deleted in source cluster`